_projects/2023/100061715/100061715.Rmd

```diff
@@ -0,0 +1,285 @@
+---
+title: "Unveiling birth clinical practice in Spain: mapping the cesarian sections and birth dynamics"
+description: |
+  "Exploring the nuances of maternal healthcare in Spain through an augmented map, showcasing original cesarean section rates alongside additional insights such as total births per autonomous community and cesarean rates adjusted for the average maternal age."
```

Comment:
BTW, remove the quotes, please. No need when the pipe `|` was specified in the first place. Also, the pipe is useful for breaking a long text into several shorter lines.